rules/rules.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package rules takes care of fetching and serving the content of Insights
+// rules (descriptions, reasons, etc.) as well as keeping the content
+// up to date by periodically updating them.
+package rules
+
+// RulesConfiguration represents the configuration of rules package
+type RulesConfiguration struct {
```

Comment:
is not it the same as rules/configuration.go?

Reply:
I was gonna use https://github.com/robfig/cron which understands the regular cron syntax, I will see how it works, thanks for the article!

Regarding the rules.go, you're right, I left two unsaved files yesterday :)